src/stretchy.js

```diff
@@ -52,130 +52,512 @@ const mathMLnode = function(label) {
+    xrightharpoondown: [0.261, 0.261, "rightharpoondown", 0.783],
+    xleftharpoondown: [0.261, 0.261, "leftharpoondown", 0.783],
+    xrightleftharpoons: [0.358, 0.358, "rightleftharpoons", 0.716],
+    xleftrightharpoons: [0.358, 0.358, "leftrightharpoons", 0.716],
+    overgroup: [0.342, 0, "overgroup", 0.87],
+    undergroup: [0.342, 0, "undergroup", 0.87],
+    xtwoheadleftarrow: [0.167, 0.167, "twoheadleftarrow", 0.86],
+    xtwoheadrightarrow: [0.167, 0.167, "twoheadrightarrow", 0.86],
+    xLongequal: [0.167, 0.167, "longequal", 0.5],
+    xtofrom: [0.264, 0.264, "tofrom", 0.86],
+};
+
+// Many of the KaTeX SVG images have been adapted from glyphs in KaTeX fonts.
+// Copyright (c) 2009-2010, Design Science, Inc. (<www.mathjax.org>)
+// Copyright (c) 2014-2017 Khan Academy (<www.khanacademy.org>)
+// Licensed under the SIL Open Font License, Version 1.1.
+// See \nhttp://scripts.sil.org/OFL
+
+// Nested SVGs
+//    Many of the KaTeX SVG images contain a nested SVG. This is done to
+//    achieve a stretchy image while avoiding distortion of arrowheads or
+//    brace corners.
+
+//    The inner SVG typically contains a very long (400 em) arrow.
+
+//    The outer SVG acts like a window that exposes only part of the inner SVG.
+//    The outer SVG will grow or shrink to match the dimensions set by CSS.
+
+//    The inner SVG always has a longer, thinner aspect ratio than the outer
+//    SVG. After the inner SVG fills 100% of the height of the outer SVG,
+//    there is a long arrow shaft left over. That left-over shaft is not shown.
+//    Instead, it is sliced off because the inner SVG is set to
+//    "preserveAspectRatio='... slice'".
+
+//    Thus, the reader sees an arrow that matches the subject matter width
+//    without distortion.
+
+//    Some functions, such as \cancel, need to vary their aspect ratio. These
+//    functions do not get the nested SVG treatment.
+
+// Second Brush Stroke
+//    Low resolution monitors struggle to display images in fine detail.
+//    So browsers apply anti-aliasing. A long straight arrow shaft therefore
+//    will sometimes appear as if it has a blurred edge.
+
+//    To mitigate this, these SVG files contain a second "brush-stroke" on the
+//    arrow shafts. That is, a second long thin rectangular SVG path has been
+//    written directly on top of each arrow shaft. This reinforcement causes
+//    some of the screen pixels to display as black instead of the anti-aliased
+//    gray pixel that a  single path would generate. So we get arrow shafts
+//    whose edges appear to be sharper.
+
+const innerSVG = {
+    // Since bcancel's SVG is inline and it omits the viewBox attribute,
+    // the browswer will render stroke-width='1.5px' in the screen context.
+    bcancel: `<line x1='0' y1='0' x2='100%' y2='100%' stroke-width='1.5px'
+stroke='currentcolor'/>`,
+
+    cancel: `<line x1='0' y1='100%' x2='100%' y2='0' stroke-width='1.5px'
+stroke='currentcolor'/>`,
```

Comment:
I originally wrote both `svgSpan` and `encloseSpan` so the the first closing `>` was in the template string, not in `innerSVG`.  But in `svgSpan`, I found that some of the SVGs were not nested, so I had to remove the `>` from of the template string and put it into `innerSVG`.

In encloseSpan, I left that closing `>` in the template string. I think it makes more sense that way.